File: Code/Web Development/eCoaching/Models/Survey/Survey.vb
<Serializable()> Public Class Survey
    Public Property ID As Integer
    Public Property LogName As String
	Public Property EmployeeID As String
	Public Property Questions As ICollection(Of Question)
    Public Property Comment As String
	Public Property HasHotTopic As Boolean
	Public Property HasPilot As Boolean
	Public Property Status As String

	Sub New()
        ID = -1
        LogName = String.Empty
		EmployeeID = String.Empty
		Questions = New List(Of Question)
        Comment = String.Empty
		HasHotTopic = False
		Status = String.Empty
		HasPilot = False
	End Sub

    Sub New(surveyID As Integer)
        ID = surveyID
    End Sub
End Class

File: Code/Web Development/eCoachingLog/eCoachingLog/Models/Survey/Survey.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace eCoachingLog.Models.Survey
{
	public class Survey
	{
		public int Id { get; set; }
		public int LogId { get; set; }
		public string LogName { get; set; }
		public string EmployeeId { get; set; }
		public IList<Question> Questions { get; set; }
		public string Comment { get; set; }
		public bool HasHotTopic { get; set; }
		public string Status { get; set; }

		public Survey()
		{
			this.Id = -1;
			this.LogName = string.Empty;
			this.Questions = new List<Question>();
			this.Comment = string.Empty;
			this.HasHotTopic = false;
			this.Status = string.Empty;
		}
	}
}